packages/yoroi-extension/app/ergo-connector/containers/ConnectWebsitesContainer.js

```diff
@@ -42,7 +44,13 @@ export default class ConnectWebsitesContainer extends Component<
     const isError = loadingWallets === LoadingWalletStates.REJECTED;
 
     if (isLoading) {
-      return <LoadingSpinner />;
+      return (
```

Comment:
The spinner appeared at the top of the page instead of being vertically centered.